src/app/_component/main/Banner.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Flex, Text } from "@radix-ui/themes";
+import Image from "next/image";
+
+export default function Banner() {
+  return (
+    <Box mx={"30px"} mt={"80px"} mb={"7%"}>
+      <Image
+        className="section-photo"
+        src="/img.svg"
+        alt="img"
+        layout="responsive"
+        width={100}
+        height={100}
+        objectFit="contain"
+        style={{ maxHeight: "500px", borderRadius: "30px" }}
+      />

```

Comment:
next.js 13이상에서는 Image 컴포넌트 업데이트로 layout, objectFit 같은 속성을 더 사용하지 않는다고 알고 있습니다. fill이나 sizes 속성을 써보는 건 어떤가요?

```suggestion
<Image
  src="/img.svg"
  alt="img"
  fill
  sizes="(max-width: 500px) 100vw, 500px"
  style={{ objectFit: "contain", maxHeight: "500px", borderRadius: "30px" }}
/>
```

관련 아티클입니다!
https://velog.io/@pixartive/%EC%99%9C-%EC%83%88%EB%A1%9C%EC%9B%8C%EC%A7%84-nextImage%EB%8A%94-%EB%8D%94%EC%9D%B4%EC%83%81-objectFit%EC%9D%84-%ED%95%84%EC%9A%94%EB%A1%9C-%ED%95%98%EC%A7%80-%EC%95%8A%EA%B2%8C-%EB%90%90%EC%9D%84%EA%B9%8C

Reply:
디테일 잡아주신 거 좋아요:)
공식문서에서도 `next/legacy/image`, `next/image` 두개로 분리하고 있고 승현님 말씀처럼 property가 빠져 있음을 두 개의 문서에서 확인하실 수 있습니다!

> Good to know: If you are using a version of Next.js prior to 13, you'll want to use the [next/legacy/image](https://nextjs.org/docs/pages/api-reference/components/image-legacy) documentation since the component was renamed.

[next/legacy/image 공식문서](https://nextjs.org/docs/pages/api-reference/components/image-legacy)
[next/image 공식문서](https://nextjs.org/docs/app/api-reference/components/image)

Reply:
두 분이 주신 문서 참고하겠습니다 감사합니다!